### DISCUSSION TITLE
Twilio Segment: pricing illustrations

### DISCUSSION THREAD

**Discussion by karoltonik** [twilio-labs/paste#3410] [Request: Illustration] [answered]
The current version of the Segment app (evergreen) utilizes illustrations (reference attached below) in the pricing sections. Do you have any recommendations on which illustrations we should use (paste migration), or do you have plans to add such illustrations?

[Figma link](https://www.figma.com/file/oeUbF1qpn2FUsmXwiTsHmN/Segment-App%3A-Paste-Migration?type=design&node-id=1372%3A33389&mode=design&t=Gw6RMIFrm2nMqSYx-1)

**Comment by loreina** [comment, marked as answer]
Hi @karoltonik,

We have some guidance on our docs site around [Illustrations](https://paste.twilio.design/foundations/illustrations#request-new-illustrations) in product UIs though the illustrations themselves are owned and managed by the Brand team. I'd suggest syncing with @kevinbtu @ceyerkes on what illustrations to swap to. Thanks!
